checklist_plus/analyzer/base.py

```
@@ -0,0 +1,371 @@
+        tokenizer = AutoTokenizer.from_pretrained(hf_ner_model)
+        model = AutoModelForTokenClassification.from_pretrained(hf_ner_model)
+
+        self.hf_ner_pipeline = pipeline("ner", model=model, tokenizer=tokenizer)
+
+    def _load_spacy_model(self):
+        """Load the spaCy model with error handling"""
+        try:
+            self.nlp = spacy.load(self.spacy_model_name)
+            logger.info(f"Loaded spaCy model: {self.spacy_model_name}")
+        except OSError as e:
+            logger.error(f"Could not load spaCy model '{self.spacy_model_name}': {e}")
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)